src/main/kotlin/no/nav/familie/ks/sak/api/dto/JournalføringDto.kt

```diff
@@ -36,12 +37,22 @@ data class JournalføringRequestDto(
     val journalførendeEnhet: String,
 )
 
-fun JournalføringRequestDto.tilOppdaterJournalpostRequestDto(sak: Sak) =
-    OppdaterJournalpostRequestDto(
+fun JournalføringRequestDto.tilOppdaterJournalpostRequestDto(
+    sak: Sak,
+    journalpost: Journalpost,
+): OppdaterJournalpostRequestDto {
+    val avsenderMottakerIdType =
+        when {
+            journalpost.kanal == "EESSI" -> journalpost.avsenderMottaker?.type
+            this.avsender.id != "" -> AvsenderMottakerIdType.FNR
+            else -> null
+        }
+
+    return OppdaterJournalpostRequestDto(
         avsenderMottaker =
```

Comment:
Disse endringene måtte inn nå grunnet oppdatering av familie-kontrakter. Se PR for PR!